lib/experimental/new_embed.dart

```diff
@@ -227,7 +227,6 @@ class NewEmbed {
     context.dartSource = gist.getFile('main.dart')?.content ?? '';
     context.testMethod = gist.getFile('test.dart')?.content ?? '';
     context.hint = gist.getFile('hint.txt')?.content ?? '';
-    showHintButton.disabled = context.hint.isEmpty;
```

Comment:
I am curious why this is going away. It seems appropriate to disable the hint button if we don’t have one

Reply:
If instructional designers don't create any hints for an exercise, I think this hint button will not show up. For disabled status, I'm not sure about use cases at the moment.

Reply:
I cut that line because I realized it was redundant. The next line sets the editor back into non-busy mode, the setter for which will disable the button if there's no hint.

Reply:
Cool, sgtm